tags/kit/context.go

```diff
@@ -0,0 +1,25 @@
+package ctx_kit
+
+import (
+	"github.com/go-kit/kit/log"
+	"github.com/grpc-ecosystem/go-grpc-middleware/logging/kit/ctxkit"
+	"golang.org/x/net/context"
+)
+
+// AddFields adds logrus fields to the logger.
+// Deprecated: should use the ctxlogrus.Extract instead
+func AddFields(ctx context.Context, fields []interface{}) {
```

Comment:
do we want to add depricated methods here? Personally this whole file can be removed instead create ctxKit directly if needed.

Reply:
Removed it in favor of ctxKit (and even fixed a bug)